raiden/api/python.py

```diff
@@ -645,6 +652,26 @@ def transfer_async(
         if not is_binary_address(target):
             raise InvalidAddress('target address is not valid.')
 
+        if secret is not None:
+            if len(secret) != 66:
+                raise InvalidSecretOrSecretHash('secret length should be 66.')
+            if not is_hex(secret):
+                raise InvalidSecretOrSecretHash('provided secret is not an hexadecimal string.')
+            secret = to_bytes(hexstr=secret)
+
+        if secret_hash is not None:
+            if len(secret_hash) != 66:
```

Comment:
and here and everywhere else below. Generally magic numbers like that are bad practise since they are error prone. Define it in a central place and then re-use it.

Reply:
Agree. Done